concept/migration/function.feature

```diff
@@ -8,16 +8,16 @@ Feature: Schema validation
   Background:
     Given typedb starts
     Given connection opens with default authentication
-    Given connection has been opened
-    Given connection does not have any database
+    Given connection is open: true
+    Given connection has 0 databases
     Given connection create database: typedb
     Given connection open schema transaction for database: typedb
 
   # TODO: Refactor existing tests to functions and add new ones!
 
 #  TODO: Refactor to functions
 #  Scenario: Types which are referenced in rules may not be renamed
-#    Given typeql define
+#    Given typeql schema query
```

Comment:
We now have:
* ``typeql schema query`` (to check if it fails or not)
* ``typeql write query`` (to check if it fails or not)
* ``typeql read query`` (to check if it fails or not)
* ``get answers of typeql schema query`` (to save the answers)
* ``get answers of typeql write query`` (to save the answers)
* ``get answers of typeql read query`` (to save the answers)

Reply:
We could probably eliminate `get answers of ...` and save the answers all the time (if the request is successful), but I decided not to change it **that** drastically =)